mantidproject/mantid
# MuonProcess no longer crops data used in the Muon Analysis GUI

The data plot settings in muon analysis were being used to crop the data in muon analysis, instead of just adjusting the plot range. This fixes that issue. 

The issue was caused by MuonProcess calling a crop routine in the background. A flag has been added (default to true so not to change the output of pre-existing scripts) to determine if the data should be cropped. 



**To test:**
1. Open Muon analysis and load some data. 
2. Go to the settings tab and change the finish value for the time axis (e.g. 10)
3. The plot should no zoom in from (about 0 to your chosen value). 
4. Change the x axis by clicking on the plot and give it the full range (0 to 32). 
5. The data should be present (it will look like a mess due to the zoom).
6. Change the time axis to custom value and then put a value in for the stat time (e.g. 5) 
7. The plot should zoom in.
8. Click the plot and look at the full data range (0 to 32). 
9. Set the finish time to 32
10. The plot should zoom in from (5 to 32)
11. Click the plot and see that the full range is present. 




Fixes #21278 . 

**Release Notes** 


---

#### Reviewer ####

Please comment on the following ([full description](http://www.mantidproject.org/Individual_Ticket_Testing)):

##### Code Review #####


##### Functional Tests #####


Does everything look good? Mark the review as **Approve**. A member of `@mantidproject/gatekeepers` will take care of it.
